src/landIce/interfaceWithMPAS/Interface.cpp

```diff
@@ -68,6 +69,10 @@ double MPAS_gravity(9.8), MPAS_rho_ice(910.0), MPAS_rho_seawater(1028.0), MPAS_s
     MPAS_ClausiusClapeyoronCoeff(9.7546e-8);
 bool MPAS_useGLP(true);
 
+bool depthIntegratedModel(false);
+
+std::vector<int> dirichletNodesIdsSepthInt;
```

Comment:
I'm assuming this was meant to be DepthInt? Fixing it might help to read the code.

Reply:
Oh, right, I'll fix this.